src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,165 @@
+package kitchenpos.application;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import kitchenpos.application.fixture.MenuFixtureFactory;
+import kitchenpos.application.fixture.MenuGroupFixtureFactory;
+import kitchenpos.application.fixture.MenuProductFixtureFactory;
+import kitchenpos.application.fixture.ProductFixtureFactory;
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.MenuProductDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Product;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+
+    @Mock
+    private MenuDao menuDao;
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    @Mock
+    private MenuProductDao menuProductDao;
+
+    @Mock
+    private ProductDao productDao;
+
+    @InjectMocks
+    private MenuService menuService;
+
+    private Menu 메뉴1;
+    private MenuGroup 메뉴그룹1;
+    private MenuProduct 메뉴상품1;
+    private MenuProduct 메뉴상품2;
+    private Product 상품1;
+    private Product 상품2;
+
+    @BeforeEach
+    void before() {
+        메뉴그룹1 = MenuGroupFixtureFactory.create(1L, "메뉴그룹1");
+        메뉴1 = MenuFixtureFactory.create(1L, "메뉴1", BigDecimal.valueOf(3000), 메뉴그룹1.getId());
+
+        상품1 = ProductFixtureFactory.create(1L, "상품1", BigDecimal.valueOf(1000));
```

Comment:
```suggestion
        황금올리브치킨 = ProductFixtureFactory.create(1L, "황금올리브치킨", BigDecimal.valueOf(20_000));
```

상품1와 같은 이름은 의미를 담지 못하는 것 같은데요.
실제 존재할 것 같은 데이터로 변경해보면 어떨까요?

이렇게 변경하면 여러 곳에서 공통으로 사용할 수 있는 Fixture로 분리할 수 있을 것 같네요 ㅎㅎ

Reply:
가독성을 챙기도록 하겠습니다!
(리뷰어님의 최애 치킨은 황금 올리브 치킨이군요!)